nano/node/transport/udp.hpp

```diff
@@ -57,6 +64,18 @@ namespace transport
 			last_packet_received = time_a;
 		}
 
+		std::chrono::steady_clock::time_point get_last_packet_sent () const
+		{
+			std::lock_guard<std::mutex> lk (channel_mutex);
+			return last_packet_sent;
+		}
+
+		void set_last_packet_sent (std::chrono::steady_clock::time_point const time_a)
```

Comment:
This can be pushed in by itself to simplify this patch.

Reply:
Now part of https://github.com/nanocurrency/nano-node/pull/1971